src/components/MainContent.js

```diff
@@ -62,45 +70,188 @@ export default function MainContent() {
+          name: "Rumi Hadid",
+          stackOrder: 2,
+          left: "left-[30px]",
+        },
+        {
+          profile: smiley,
+          name: "Jane Doe",
+          stackOrder: 3,
+          left: "left-[45px]",
+        },
+      ],
+      budget: "$14,000",
+      completion: 60,
+    },
+    {
+      logo: letterlogo,
+      project: "Add Progress Track",
+      members: [
+        {
+          profile: smiley,
+          name: "Rumi Hadid",
+          stackOrder: 0,
+          left: "left-[0px]",
+        },
+        {
+          profile: smiley,
+          name: "Jessica Doe",
+          stackOrder: 1,
+          left: "left-[15px]",
+        },
+      ],
+      budget: "$3,000",
+      completion: 10,
+    },
+    {
+      logo: slacklogo,
+      project: "Fix Platform Errors",
+      members: [
+        {
+          profile: smiley,
+          name: "Marissa Ro",
+          stackOrder: 0,
+          left: "left-[0px]",
+        },
+        {
+          profile: smiley,
+          name: "Jane Doe",
+          stackOrder: 1,
+          left: "left-[15px]",
+        },
+      ],
+      budget: "Not set",
+      completion: 100,
+    },
+    {
+      logo: spotifylogo,
+      project: "Launch our Mobile App",
+      members: [
+        {
+          profile: smiley,
+          name: "Jane Dems",
+          stackOrder: 0,
+          left: "left-[0px]",
+        },
+        {
+          profile: smiley,
+          name: "Rick Wills",
+          stackOrder: 1,
+          left: "left-[15px]",
+        },
+        {
+          profile: smiley,
+          name: "Jen Akins",
+          stackOrder: 2,
+          left: "left-[30px]",
+        },
+        {
+          profile: smiley,
+          name: "Stuart Little",
+          stackOrder: 3,
+          left: "left-[45px]",
+        },
+      ],
+      budget: "$20,500",
+      completion: 100,
+    },
+    {
+      logo: bluelogo,
+      project: "Add the New Pricing Page",
+      members: [
+        {
+          profile: smiley,
+          name: "Ron Doe",
+          stackOrder: 0,
+          left: "left-[0px]",
+        },
+      ],
+      budget: "$500",
+      completion: 25,
+    },
+    {
+      logo: redsquare,
+      project: "Redesign New Online Shop",
+      members: [
+        {
+          profile: smiley,
+          name: "Jane Wane",
+          stackOrder: 0,
+          left: "left-[0px]",
+        },
+        {
+          profile: smiley,
+          name: "Tiger Woods",
+          stackOrder: 1,
+          left: "left-[15px]",
+        },
+      ],
+      budget: "$2,000",
+      completion: 40,
+    },
+  ];
+
   return (
-    <div className="box w-screen h-screen grid grid-cols-2 overflow-y-auto bg-gray-200">
+    <div className="w-screen h-screen grid grid-cols-2 bg-gray-100 overflow-y-auto">
       <div className="w-[calc(100vw-238px)] h-full ml-[218px] grid grid-cols-6 auto-rows-min gap-5">
         <MainNavBar />
+
         {/* Company info section */}
```

Comment:
You can make each of these comments and code below it a component of its own..
<CompanyInfo>